radiography/src/main/java/radiography/ViewStateRenderers.kt

```diff
@@ -72,11 +71,10 @@ public object ViewStateRenderers {
 
       // Semantics
       composeView
-        .modifiers
-        .filterIsInstance<SemanticsModifier>()
+        .semanticsConfigurations
```

Comment:
this, and the change in [Semantics.kt](https://github.com/square/radiography/pull/169/files#diff-2b27936776ce7e7111731ed2ede9ca41c137ef09803bad6d1a7a99f9dd8dac7b), are the main changes to support 1.7, where we need to use the semantics nodes instead of the modifiers